googleMapsApiModule.py

```diff
@@ -27,3 +27,13 @@ def mapsstatic(search_location):
     result_url = f'https://maps.googleapis.com/maps/api/staticmap?center={address.lower().replace(" ", "+")}&zoom=13&scale=1&size=600x350&maptype=roadmap&key={config.key}&format=png&visual_refresh=true&markers=size:mid%7Ccolor:0xff0000%7Clabel:L%7C{address.lower().replace(" ", "+")}'
     print(result_url)
     webbrowser.open_new(result_url)
+
+def elevation(search_location):
+    result = gmaps.geocode(search_location)
+    json = requests.get(f'https://maps.googleapis.com/maps/api/elevation/json?locations={result[0]["geometry"]["location"]["lat"]},{result[0]["geometry"]["location"]["lng"]}&key={config.key}').json()
```

Comment:
@vishakha-lall Should we move all urls to the `constants.py` file and import them from there? I think I missed that PR so I'm still trying to understand the logic behind moving only some of the urls to the file.

Reply:
@janakrajchadha the URLs currently in `constants.py` have a common format, after which different parameters are added to them for different APIs. `MapsStatic` API doesn't have such a URL format. In the case of `Elevation` API, I couldn't test the URL due to error #62. Once the bug is cleared, I'll test and add the URL in `constants.py`.

Reply:
I'll add the URL and also do some more formatting in other functions as well, in a different PR, if that's okay.

Reply:
@adityabisoi we'll take a look at that separately.